src/bp-core/bp-core-functions.php

```diff
@@ -5286,3 +5286,14 @@ function bp_is_admin( $screen_id = '' ) {
 
 	return $screen_id === $bp->admin->current_screen;
 }
+
+/**
+ * Common function to check if bp_classic is active or not.
+ *
+ * @since 15.0.0
+ *
+ * @return bool True if bp_classic is active, false otherwise.
+ */
+function bp_is_classic() {
+	return function_exists( 'bp_classic' ) || is_plugin_active( 'bp-classic/bp-classic.php' );
```

Comment:
All PHP Unit tests are failing, we need more work about this PR. For a reason I ignore for now the `is_plugin_active()` function is not yet available when doing these tests.

`PHP Fatal error:  Uncaught Error: Call to undefined function is_plugin_active() in /tmp/wordpress/wp-content/plugins/buddypress/src/bp-core/bp-core-functions.php:5298`

Reply:
@imath Thanks for the review and feedback, should we add something like this to `tests/phpunit/bootstrap.php` to handle that fatal error?

`if ( ! function_exists( 'is_plugin_active' ) ) {
    require_once ABSPATH . 'wp-admin/includes/plugin.php';
}
`

Reply:
I _think_ `is_plugin_active` is not loaded for front-end requests, only `wp-admin` requests. So when we load `bp-nouveau`, we make use of this function and it is complaining it doesn't exist.

```php
if ( ! function_exists( 'is_plugin_active' ) ) {
  require_once ABSPATH . 'wp-admin/includes/plugin.php';
} 
```

^ this seems to be the solution suggested [here](https://developer.wordpress.org/reference/functions/is_plugin_active/#comment-616) for front-end support too.